include/sys/kasan.h

```diff
@@ -53,6 +53,9 @@ typedef struct {
  * memory is usable. */
 void init_kasan(void);
 
+/* Increase address space covered by KASAN by size. */
+void kasan_grow(size_t size);
```

Comment:
Perhaps passing a virtual address instead of size would yield a cleaner interface? It would be more consistent with `pmap_growkernel()`, and we would no longer need for the extra `size` variable in `pmap_growkernel()`.

Reply:
done